src/Gui/Navigation/NavigationStyle.cpp

```diff
@@ -1164,6 +1156,20 @@ void NavigationStyle::spin_simplified(SbVec2f curpos, SbVec2f prevpos)
 {
     assert(this->spinprojector);
 
+    if (getOrbitStyle() == FreeTurntable) {
+        SbVec2f midpos(prevpos[0], curpos[1]);
+        spin_simplified_internal(curpos, midpos);
+        spin_simplified_internal(midpos, prevpos);
+    }
+    else {
+        spin_simplified_internal(curpos, prevpos);
+    }
+
+    hasDragged = true;
+}
+
+void NavigationStyle::spin_simplified_internal(SbVec2f curpos, SbVec2f prevpos)
+{
```

Comment:
I see I missed this review on the last go-round for `spin_simplified` -- nevertheless, to try not to lose any more consistency,
```suggestion
    if (getOrbitStyle() == FreeTurntable) {
        SbVec2f midpos(prevpos[0], curpos[1]);
        spinSimplifiedInternal(curpos, midpos);
        spinSimplifiedInternal(midpos, prevpos);
    }
    else {
        spinSimplifiedInternal(curpos, prevpos);
    }

    hasDragged = true;
}

void NavigationStyle::spinSimplifiedInternal(SbVec2f curpos, SbVec2f prevpos)
{
```